src/main/java/uk/gov/hmcts/divorce/common/notification/Applicant1AppliedForFinalOrderNotification.java

```diff
@@ -51,26 +54,47 @@ public void sendToApplicant1(final CaseData caseData, final Long id) {
 
     @Override
     public void sendToApplicant2Solicitor(CaseData caseData, Long caseId) {
-        if (!caseData.getApplicationType().isSole()
-                && YES.equals(caseData.getFinalOrder().getApplicant1AppliedForFinalOrderFirst())) {
-            log.info("Sending Applicant 2 notification informing them that other party have applied for final order: {}", caseId);
-            notificationService.sendEmail(
-                caseData.getApplicant2().getSolicitor().getEmail(),
-                JOINT_SOLICITOR_OTHER_PARTY_APPLIED_FOR_FINAL_ORDER,
-                commonContent.solicitorTemplateVars(caseData, caseId, caseData.getApplicant2()),
-                ENGLISH
-            );
+        if (!caseData.getApplicationType().isSole()) {
+            if (Objects.isNull(caseData.getFinalOrder().getApplicant2AppliedForFinalOrder())) {
+                log.info("Sending Applicant 2 notification informing them that other party have applied for final order: {}", caseId);
+                notificationService.sendEmail(
+                        caseData.getApplicant2().getSolicitor().getEmail(),
+                        JOINT_SOLICITOR_OTHER_PARTY_APPLIED_FOR_FINAL_ORDER,
+                        commonContent.solicitorTemplateVars(caseData, caseId, caseData.getApplicant2()),
+                        caseData.getApplicant2().getLanguagePreference()
+                );
+            } else {
+                sendBothSolicitorsAppliedForFinalOrderNotification(caseData, caseId, caseData.getApplicant2());
+            }
+        }
+    }
+
+    @Override
+    public void sendToApplicant1Solicitor(CaseData caseData, Long caseId) {
+        if (!caseData.getApplicationType().isSole() && Objects.nonNull(caseData.getFinalOrder().getApplicant1AppliedForFinalOrder())) {
```

Comment:
i think this check should be for applicant 2
> Objects.nonNull(caseData.getFinalOrder().getApplicant1AppliedForFinalOrder())

Reply:
Reverted the change as spotted by Sunil